src/page/ConfigPageLayout/tabs/SecretsManagementTab/SecretEditModal.test.tsx

```diff
@@ -0,0 +1,273 @@
+import React from 'react';
+import { screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { DataTestIds } from 'test/dataTestIds';
+import { MOCKED_SECRETS_API_RESPONSE } from 'test/fixtures/secrets';
+import { apiRoute, getServerRequests } from 'test/handlers';
+import { render as testRender } from 'test/render';
+import { server } from 'test/server';
+
+import { SECRETS_EDIT_MODE_ADD } from './constants';
+import { SecretEditModal } from './SecretEditModal';
+
+async function render(element: React.ReactElement) {
+  const result = testRender(<div data-testid={DataTestIds.CONFIG_CONTENT}>{element}</div>);
+  await waitFor(() => expect(screen.getByTestId(DataTestIds.CONFIG_CONTENT)).toBeInTheDocument(), {
+    timeout: 10000,
```

Comment:
Seems like a pretty high number for a timeout, is it really needed?

Reply:
When running only tests for that file, 30ms is enough (on my machine), when running all the tests, 40ms seems to be the sweet spot.

If I put the CPU under load while running all the tests, there are situations where a timeout of 10s is too little (`CheckList.test.tsx` has a timeout of 10s, and it manages to fail under heavy load).

I don't mind setting it at a "reasonable" value, but it becomes more likely that slow running test (CI etc) will fail.